data-sources/pagerduty/config.yml

```diff
@@ -0,0 +1,9 @@
+id: pagerduty
```

Comment:
Just links to notification channels docs. Notification channels are on the way out in favor workflows. Is this worth making a data source?

Do we just accept this one now and update install target later?